### DISCUSSION TITLE
Want to help/contribute

### DISCUSSION THREAD

**Discussion by ajmalafif** [d01000100/figma-token-engine#43] [Q&A] [answered]
hey there @d01000100 

sorry to put this here, I didn‘t see the Discussion tab, so feel free to close this if this is a wrong place. I want to ask if there‘s a need and if yes the priority of having a end-to-end demo of how this works?

I mainly ask for my own usage but I don‘t mind tinkering and figuring out myself, then help out with a video tutorial/demo if you‘re open to it. 

Thanks for open sourcing this, I definitely have a use case for this but haven‘t yet figure out how to integrate in our workflow

**Comment by d01000100** [comment]
I was just talking to my team and I'll be working this week on a media post (medium/dev.to) doing a step-by-step guide on how to install, configure and use the figma token engine ¿is that what you were thinking about? ¿Or were you thinking on something inside the repo?

**Reply by ajmalafif** [reply]
sorry I saw the email and couldn‘t find the issue (🤦🏻‍♂️) and thought it was closed (non-issue). But you did everything right (the step-by-step guide) and move it too Discussion tab 💥🙇🏻‍♂️

Thank you!

**Comment by d01000100** [comment, marked as answer]
I just published a [Quick Guide](https://medium.com/@jdanielca/figma-token-engine-quick-start-b6e0bc08a388) showing how to configure and use the service! Hope it helps. The documentation does need more details, which will be on my list of priorities.

**Reply by ajmalafif** [reply]
Thanks so much for this, will go through and surface any feedback I have (if any)
